src/components/ErrorBoundary.js

```diff
@@ -1,31 +1,58 @@
 import React from "react";
 import Alert from "@material-ui/lab/Alert";
 
-function defaultFormatter(error) {
+/**
+ * A basic FallbackComponent for the {@link ErrorBoundary}. Displays a "yikes"
+ * message using a Material UI Alert.
+ */
+export function BasicError(props) {
+  // Cleanup the message to prevent potential weird spacing around the parenthesis
+  const message = props.error.message?.trim();
   return (
     <Alert severity="error">
-      Yikes! Something went wrong loading the OSCAL data. Sorry, we&apos;ll look
-      into it. ({error.message})
+      Yikes! Something went wrong loading the OSCAL data. ({message})
```

Comment:
@rgauss I kept this simple. But do you have any thoughts on:

```suggestion
      <details>
        <summary>Yikes! Something went wrong loading the OSCAL data.</summary>
        {message}
      </details>
```

It'd look a little like:

![image](https://user-images.githubusercontent.com/850893/156170269-281777d2-41d3-46ea-808c-606408226a99.png)

![image](https://user-images.githubusercontent.com/850893/156170317-92aff2bd-41b8-4356-8c5f-c9e1c4901f25.png)

![image](https://user-images.githubusercontent.com/850893/156170375-b4389189-c160-4e72-b694-aae8fb43ab29.png)

![image](https://user-images.githubusercontent.com/850893/156170442-a0ba8f05-04ca-43ff-95d4-1aee29a1a493.png)

![image](https://user-images.githubusercontent.com/850893/156170489-423aec29-c24f-45ab-a6f6-e84add97c61b.png)

Reply:
Sure, that looks good.